js/fileUtils.ts

```diff
@@ -105,6 +105,18 @@ export async function promptForFileReferences(
   });
 }
 
+export async function promptForUrlReference(): Promise<Track> {
```

Comment:
Does this need to be async? `prompt` is synchronous. Is the idea to make it async for a future where we need to do some async work?